src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -137,9 +137,14 @@ private function getContainerCompilerLogs()
 
         $logs = array();
         foreach (file($file, FILE_IGNORE_NEW_LINES) as $log) {
-            $log = explode(': ', $log, 2);
+            if (false !== ($pos = strpos($log, ':')) && $pos !== (strlen($log) - 1)) {
+                list($pass, $message) = explode(': ', $log, 2);
+            } else {
+                $pass = 'Unknown Compiler Pass';
```

Comment:
@nicolas-grekas this seems flaky due to disabling autoloading on the class_exists (becomes unknown). As this logging is aggregated during a run-time request, I can imagine the classes to never exist in the first place.

It works fine if I allow autoloading and I don't think this should be an issue for dev